src/components/DialoguesWithoutCuratorButton.tsx

```diff
@@ -0,0 +1,45 @@
+import { ChatBubble } from '@mui/icons-material';
+import { Box, Button } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { TicketSearchParam } from '../api/searchApi';
+import { TicketStatus } from '../types/publication_types/ticket.types';
+import { dataTestId } from '../utils/dataTestIds';
+
+export const DialoguesWithoutCuratorButton = () => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const searchParams = new URLSearchParams(history.location.search);
+  const statusNew: TicketStatus = 'New';
+
+  const toggleDialoguesWithoutCurators = () => {
+    if (dialoguesWithoutCuratorSelected()) {
+      searchParams.delete(TicketSearchParam.Status);
+    } else {
+      searchParams.set(TicketSearchParam.Status, statusNew);
+      searchParams.delete(TicketSearchParam.Assignee);
+    }
+    history.push({ search: searchParams.toString() });
+  };
+
+  const dialoguesWithoutCuratorSelected = () => {
+    return searchParams.get(TicketSearchParam.Status) === statusNew && !searchParams.get(TicketSearchParam.Assignee);
+  };
```

Comment:
trenger ikke funksjon her
```suggestion
  const dialoguesWithoutCuratorSelected = 
    return searchParams.get(TicketSearchParam.Status) === statusNew && !searchParams.get(TicketSearchParam.Assignee);
```